### PR TITLE
Port psapi to new winapi version

### PR DESCRIPTION
**Pull request by GuillaumeGomez**

### CODE REVIEW

File: src/um/psapi.rs [hunk 1]
```diff
@@ -0,0 +1,232 @@
...
+use shared::minwindef::{BOOL, DWORD, LPDWORD, PDWORD, HMODULE, LPVOID};
+use um::winnt::{LPWSTR, LPCWSTR, LPSTR, LPCSTR, HANDLE, PVOID};
+
+pub type LPMODULEINFO = *mut MODULEINFO;
+pub type PPSAPI_WORKING_SET_INFORMATION = *mut PSAPI_WORKING_SET_INFORMATION;
+pub type PPSAPI_WORKING_SET_EX_INFORMATION = *mut PSAPI_WORKING_SET_EX_INFORMATION;
+pub type PPSAPI_WS_WATCH_INFORMATION = *mut PSAPI_WS_WATCH_INFORMATION;
+pub type PPSAPI_WS_WATCH_INFORMATION_EX = *mut PSAPI_WS_WATCH_INFORMATION_EX;
+pub type PENUM_PAGE_FILE_CALLBACKA = Option<unsafe extern "system" fn(pContext: LPVOID, pPageFileInfo: PENUM_PAGE_FILE_INFORMATION, lpFilename: LPCSTR) -> BOOL>;
+pub type PENUM_PAGE_FILE_CALLBACKW = Option<unsafe  extern "system" fn(pContext: LPVOID, pPageFileInfo: PENUM_PAGE_FILE_INFORMATION, lpFilename: LPCWSTR) -> BOOL>;
```
Comment (retep998) [inline src/um/psapi.rs]: Use `FN!` for function pointers.

**Comment by retep998** [comment]
PR has conflicts.

**Comment by GuillaumeGomez** [comment]
Not anymore.

File: src/um/psapi.rs [hunk 2]
```diff
@@ -0,0 +1,241 @@
+// Copyright © 2016-2017 winapi-rs developers
+// Licensed under the Apache License, Version 2.0
+// <LICENSE-APACHE or http://www.apache.org/licenses/LICENSE-2.0> or the MIT license
+// <LICENSE-MIT or http://opensource.org/licenses/MIT>, at your option.
+// All files in the project carrying such notice may not be copied, modified, or distributed
+// except according to those terms.
+//! FFI bindings to psapi.
+
+use shared::basetsd::{ULONG_PTR, SIZE_T};
```
Comment (retep998) [inline src/um/psapi.rs]: Alphabetical order for imports.

Comment (GuillaumeGomez) [inline src/um/psapi.rs]: Yes sir!

**Pull request merged**